.ruby-version

```diff
@@ -1 +1 @@
-3.3.0
+3.1.1
```

Comment:
Downgrading to the same version that is running in the docker compose container.